DP3TApp/Logic/AppDelegate.swift

```diff
@@ -120,10 +129,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             shouldJump = UIStateManager.shared.uiState.shouldStartAtReportsDetail && UIStateManager.shared.uiState.reportsDetail.showReportWithAnimation
         }
         if shouldJump,
-            let navigationController = window?.rootViewController as? NSNavigationController,
-            let homescreenVC = navigationController.viewControllers.first as? NSHomescreenViewController {
+            let tabBarController = tabBarController {
+            tabBarController.currentTab = .homescreen
+
+            let navigationController = tabBarController.currentNavigationController
```

Comment:
Now that the tab bar view controller lives inside the navigation controller, this can be reverted again.